lib/tools/broccoli/broccoli-plugin-wrapper.ts

```diff
@@ -0,0 +1,59 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+
+import fs = require('fs');
+import path = require('path');
+import {TreeDiffer} from './tree-differ';
+
+export class BroccoliPluginWrapper implements BroccoliTree {
+  treeDiffer: TreeDiffer;
+  wrappedPlugin: IBroccoliPlugin = null;
+  inputTree: any = null;
+  description: string = null;
+  absoluteOutputPath: string = null;
+  treeRootDirName: string = null;
+  projectDir: string = null;
+  $injector: IInjector = null;
+
+  // props monkey-patched by broccoli builder:
+  inputPath: string = null;
+  cachePath: string = null;
+  outputPath: string = null;
+
+  constructor(private pluginClass: any, wrappedPluginArguments: any) { 
+      this.inputTree = wrappedPluginArguments[0];
+      this.description = this.pluginClass.name;
+      this.absoluteOutputPath = wrappedPluginArguments[1];
+      this.treeRootDirName = wrappedPluginArguments[2];
+      this.projectDir = wrappedPluginArguments[3];
+      this.$injector = $injector.resolve("injector");
+  }
+
+  rebuild(): any {
```

Comment:
The code does not return but function is declared to return any. Consider declaring it as void.